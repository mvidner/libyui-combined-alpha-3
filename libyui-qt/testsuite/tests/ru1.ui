
# ---------------------------------------------------------
#
#  Filename:    Ru1.ui
#
#  Purpose:     test cases for ui test
#
#  Creator:     jd@suse.de
#
#  Maintainer:  jd@suse.de
#
#  Remarks:     Always the user interaction is removed:
#                   any answer = UserInput();
#               Furthermore the tested action is set into a loop
#
# ---------------------------------------------------------

{  
    SetCharset("ISO 8859-5");
    any ok = `HBox(`Stretch(), `Pushbutton(`Id("ok"), "OK"), `Stretch());
    any vbox = `VBox( 

// Main.cpp:112
    `Label ("Copying system config file."),
    `Label ( "Копирование Системной Конфугурации"),

// Main.cpp:231 Main.cpp:250 MenuAction.cpp:364 MenuAction.cpp:386 MenuAction.cpp:878
    `Label ("DISMISS"),
    `Label ( "ПРОДОЛЖИТЬ"),

// Main.cpp:235 MenuAction.cpp:371
    `Label ("Yudit: information"),
    `Label ( "Yudit: информация"),

// Main.cpp:254
    `Label ("Yudit: error"),
    `Label ( "Yudit: ошибка"),

// MainWidget.cpp:208
    `Label ("File"),
    `Label ( "Файл"),

// MainWidget.cpp:210
    `Label ("Open..."),
    `Label ( "Открыть..."),

// MainWidget.cpp:212
    `Label ("Append..."),
    `Label ( "Добавить..."),

// MainWidget.cpp:214
    `Label ("Re-Load"),
    `Label ( "Перезагрузить"),

// MainWidget.cpp:216
    `Label ("New Window"),
    `Label ( "Новое Окно"),

// MainWidget.cpp:219
    `Label ("Save"),
    `Label ( "Записать"),

// MainWidget.cpp:221
    `Label ("Save As..."),
    `Label ( "Записать В Файл..."),

// MainWidget.cpp:224 MainWidget.cpp:317
    `Label ("Config"),
    `Label ( "Конфигурация"),

// MainWidget.cpp:226
    `Label ("Print..."),
    `Label ( "Печать..."),

// MainWidget.cpp:229
    `Label ("Close"),
    `Label ( "Закрыть"),

// MainWidget.cpp:231
    `Label ("Exit All"),
    `Label ( "Конец Работы"),
      ok
     );

    integer loop = 0;
    while( loop < 10 )
    {
        ShowDialog( Dialog( vbox ));
        any CurrentMap = QueryDialog( );
        CloseDialog();

        if( ( is( CurrentMap, map ) == false ) || 
            ( size( CurrentMap ) != 0 ) ) return false;

        loop = loop + 1;
    }
    return(  "If you get a warning next, something like" +
              "WARNING: No font available with requested character set! #1" +
              "please have a look at the installed fonts." +
              "At the moment variable fonts are not installed"  );
}

