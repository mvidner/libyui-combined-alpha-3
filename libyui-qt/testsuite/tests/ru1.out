(`stdio())
("If you get a warning next, something likeWARNING: No font available with requested character set! #1please have a look at the installed fonts.At the moment variable fonts are not installed")
